Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -116,7 +116,7 @@ class OrderedNamedLinesCollector {
 
 protected:
 
-    enum NamedLinesType { NamedLines, AutoRepeatNamedLines };
+    enum class NamedLinesType { NamedLines, AutoRepeatNamedLines };
```

Comment:
We prefer specifying underlying type like:
`enum class NamedLinesType : bool { NamedLines, AutoRepeatNamedLines };`